article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -98,10 +99,25 @@ object ArticlePicker {
 
     if (tier == RemoteRender) {
       DotcomponentsLogger.logger.logRequest(s"path executing in dotcomponents", features, page)
+    } else if (tier == PressedArticle) {
+      DotcomponentsLogger.logger.logRequest(s"path executing from pressed content", features, page)
     } else {
       DotcomponentsLogger.logger.logRequest(s"path executing in web", features, page)
     }
 
     tier
   }
+
+  def decideTier(isPressed: Boolean, dcrCanRender: Boolean)(implicit
+      request: RequestHeader,
+  ): RenderType = {
+    if (request.forceDCROff) LocalRenderArticle
+    else if (isPressed) PressedArticle
```

Comment:
Question: If I pass a url for a pressed article with the flag `?dcr=false` should I still get the Pressed version or the Frontend version?

Reply:
If we still want to serve pressed content regardless the flag then we should maybe swap these lines?

```suggestion
    if (isPressed) PressedArticle
    else if (request.forceDCROff) LocalRenderArticle
```

Reply:
If passing the query param `?dcr-false` we should get the frontend rendered version. We want forceDCROff to be prioritised over pressing in the following situations:
- we want to validate how the article renders in frontend, e.g. why was this article pressed and are you sure the pressed version matches what we previously had in frontend?
- we want/need to re-press the article, e.g. the article owner updated the page content and we need to re-press.

I'd suggest leaving the order in this case.

Reply:
_Ideally_ there would be a `?pressed=false` flag we could use...